deno.json

```diff
@@ -30,7 +30,8 @@
     "feed": "https://esm.sh/feed@4.2.2",
     "kv_oauth": "https://deno.land/x/deno_kv_oauth@v0.2.5/mod.ts",
     "@twind/core": "https://esm.sh/@twind/core@1.1.3",
-    "chart.js": "https://esm.sh/stable/chart.js@4.0.1/auto"
+    "chart.js": "https://esm.sh/chart.js@4.3.0",
```

Comment:
even though the import for the types via `jsdelivr` has been working flawlessly in the `fresh_charts` repo, the pipeline kept on failing here.

will need to come up w/ a fix for the underlying issue asap.